github/ci/prow/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -300,31 +300,75 @@ presubmits:
           requests:
             memory: "15Gi"
 
-  - name: check-up-kind-k8s-sriov-1.14.2
+  - name: check-up-kind-1.17-sriov
+    annotations:
+      k8s.v1.cni.cncf.io/networks: multus-cni-ns/sriov-passthrough-cni
     always_run: false
```

Comment:
should be true

Reply:
This job has to be run manually since it uses a sriov node, we don't want it to run always at any kubevirtci change.

Reply:
can we address it in a future PR, which let prow select the nodes that can run it ?
else we would forget to run it / it would break from time to time (unless we at least have some cron job)